engine/api/broadcast/broadcast.go

```diff
@@ -0,0 +1,88 @@
+package broadcast
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/sdk"
+)
+
+// InsertBroadcast insert a new worker broadcast in database
+func InsertBroadcast(db gorp.SqlExecutor, bc *sdk.Broadcast) error {
```

Comment:
if it's not too late:
`InsertBroadcast` should be `Insert` because `broadcast.InsertBroadcast` is annoying. `broadcast.Insert` is better